core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -437,8 +437,13 @@ angular.module('oppia').directive('storyNodeEditor', [
               StoryEditorStateService.onStoryReinitialized.subscribe(
                 () => _init()
               ));
-            $scope.$on('recalculateAvailableNodes', _recalculateAvailableNodes);
-
+            ctrl.directiveSubscriptions.add(
+              StoryEditorStateService.onRecalculateAvailableNodes.subscribe(
+                () => {
+                  _recalculateAvailableNodes();
```

Comment:
May be move this to one line itself

() => _recalculateAvailableNode()

Reply:
Done!